### ISSUE TITLE
Unable to open OSX release 3.0.6 due to outdated NODE version error

### ISSUE THREAD

**Issue by BigslimVdub** [arqma/arqma-electron-wallet#66]
```Uncaught Exception:
Error: The module '/Applications/Arqma Electron Wallet.app/Contents/Resources/app.asar.unpacked/node_modules/zeromq/build/Release/zmq.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 67. This version of Node.js requires
NODE_MODULE_VERSION 69. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).```

**Comment by marko-lorentz** [comment]
I stumbled across this too on May, 13th.
The only solution I found was: https://stackoverflow.com/questions/46384591/node-was-compiled-against-a-different-node-js-version-using-node-module-versio#answer-52796884

(Basically a big rebuild of everything including some native parts so it requires a C compiler from homebrew or Xcode.)

**Comment by BigslimVdub** [comment]
I am unable to even build on Catalina. I used to be able to compile on high Sierra no issues with no changes on my end.

**Comment by BigslimVdub** [comment]
Ok so I figured out the build issue, apparently upgrading to Catalina decided to wipe xcode and some brew installs from my system. I reinstalled Xcode, fixed brew, and was able to compile after running ``npm rebuild`` with the same error code for the app when executing. Node version 11 and 11.9.0. I updated to 15.14.0 and ran ``npm rebuild`` and now get the error zmq not supported on x64? I think electron 4.2 does not like Catalina. 

```Uncaught Exception:
Error: No native build was found for platform=darwin arch=x64 runtime=electron abi=69 uv=1 libc=glibc node=10.11.0 electron=4.2.12```

**Comment by marko-lorentz** [comment]
(Me using Mac OS "Big Sur" 11.3.1. I had to use `nvm` to force the Node version to 11.9.0 IIRC. The link I posted 5 days ago was very helpful to push thing further ... but ... as you mention it: ZMQ needed some extra treatment. Maybe `cd` to its folder and force a dedicated rebuild? Sorry - you should have taken some notes.)

**Comment by BigslimVdub** [comment]
The issue was addressed with @muscleman repo [updates](https://github.com/muscleman/arqma-electron-wallet/commit/e029795604e5ea6fb7ede428181ae667b28adcc1) and required using 12.20.2 Node version along with ARQMA 6.0.2 binaries. 
I have built working binaries for Catalina here: https://github.com/BigslimVdub/arqma-electron-wallet/releases/tag/3.0.7_BigSlim 
sha256 can be checked as needed

The above fixes were merged to main it appears on #69

**Comment by marko-lorentz** [comment]
Can confirm for Mac OS 11.3.1:
* Set Node to 12.20.0 via `nvm`
* Needed `npm --force install` to overwrite old symlinks
* Needed `npm rebuild node-sass`
--> Dev version works
--> Executable in `dist/electron-mat/Packaged/mac/Arqma\ Electron\ Wallet.app/Contents/MacOS/Arqma\ Electron\ Wallet` works

(Use `xattr -d com.apple.quarantine bin/arqmad` and `xattr -d com.apple.quarantine bin/arqma-wallet-rpc` when Mac OS complains that these executable come from an untrusted source.)
